lib/app/store/user.js

```diff
@@ -1,8 +1,32 @@
+const got = require('got')
 const INIT = { credentials: 'include', headers: { accept: 'application/json' } }
 
 module.exports = function user () {
   return (state, emitter) => {
     state.user = state.user || {}
+    state.user.notificationsAmount = -1
+
+    var rerender = () => emitter.emit('render')
+
+    if (state.user.isAuthenticated) {
+      let requestOptions = {retry: 0,
+        headers: {brfauth: state.user.forumAuthenticationToken, 'Cache-Control': 'no-store'}}
+
+      got(process.env.FORUM_URL + '/api/unread', requestOptions)
```

Comment:
I assume this is only used in the client? If so it could easily be switched out for `window.fetch` and we wouldn't have to include the entire got library in the client bundle. `window.fetch` is also polyfilled with polypill.io so there are no compatibility issues with using it.